distributed-calculator/react-calculator/package.json

```diff
@@ -11,7 +11,7 @@
     "dependencies": {
         "body-parser": "^1.18.3",
         "express": "^4.16.4",
-        "request": "^2.88.0"
+        "axios": "^0.25.0"
```

Comment:
nit: v0.26.0 is out, should we upgrade?

Reply:
Good idea Shubham, I'm putting it and if tests look good, let's ship it!  thank you